scripts/BilbyCustomFunctions_Reduction.py

```diff
@@ -90,22 +93,24 @@ def files_to_reduce(parameters, evaluate_files):
     if len(evaluate_files) == 0:
         files_to_reduce.extend(parameters)
     else:
-        evaluate_files_l = evaluate_files_list(evaluate_files)  # call funciton for retrieve the IDs list
+        evaluate_files_l = evaluate_files_list(evaluate_files)  # call function for retrieve the IDs list
         for parameter in parameters:
             if int(parameter['index']) in evaluate_files_l:
                 files_to_reduce.append(parameter)
 
     return files_to_reduce
 
+##############################################################################
+# For output file formatting #################################################
```

Comment:
Please use Python docstrings for documenting functions. This keeps documentation consistent.

(the `###` lines between functions are bad practice too, however they are already used in this file so I have no great opinion about removing them)